src/sagemaker_containers/environment.py

```diff
@@ -568,7 +580,8 @@ def create(cls, session=None):  # type: (boto3.Session) -> Environment
 
         input_data_config = read_input_data_config()
 
-        sagemaker_hyperparameters, hyperparameters = split_hyperparameters(read_hyperparameters())
+        hps = read_hyperparameters()
```

Comment:
hyperparameters

Reply:
That name exists already https://github.com/mvsusp/sagemaker-containers/blob/597d9b0319e1c64b57a438df9417eca49e3e3e64/src/sagemaker_containers/environment.py#L420

Reply:
ya but we should take opportunity to fix

Reply:
Should I move the Environment class to another file them? 
What do you suggest @jesterhazy